packages/generate-en10168-pdf-template/src/lib/createInspection.ts

```diff
@@ -21,7 +22,22 @@ import { PRODUCT_DESCRIPTION_COLUMNS_COUNT } from './constants';
 import { renderMeasurement, renderMeasurementArray } from './measurement';
 import { supplementaryInformation } from './supplementaryInformation';
 
-export function createInspection(inspection: Inspection, i18n: I18N): (TableElement | ContentText | ContentCanvas)[] {
+export function createInspection(inspection: Inspection, i18n: I18N): InspectionUnionType {
```

Comment:
As said earlier, the inspection can be undefined now, and that case would not be handled by the generator.
Could you cover that case and add unit test please?

Reply:
I updated `template.hbs` and added `valid_certificate_5.json`